openc3-cosmos-cmd-tlm-api/app/controllers/script_autocomplete_controller.rb

```diff
@@ -30,23 +30,38 @@ class ScriptAutocompleteController < ApplicationController
                     limits_enabled? enable_limits disable_limits
                     check check_tolerance wait wait_tolerance wait_check wait_check_tolerance)
 
-  def get_reserved_item_names
+  def target_packet_items
```

Comment:
This method is way too expensive.  It returns every item for every target in existence.

Reply:
Also doesn't appear to be used?

Reply:
Used by the autocomplete via the following route: `get '/autocomplete/target-packet-items', to: 'script_autocomplete#target_packet_items'`